lang_python.adoc

```diff
@@ -13,130 +13,171 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
 `-t python` option:
 
 [source,shell]
-kaitai-struct-compiler -t python my_spec.ksy
+kaitai-struct-compiler -t python --python-package . my_spec.ksy
 
 This normally generates `my_spec.py` parser module, which is compatible
-with both Python 2 and 3.
+with both Python 2 and 3. It is strongly recommended to use Python 3
+because Python 2 is EOL since 2020-01-01. You can also install
+`python-is-python3` on Ubuntu focal to make `python3` be used via
+`python` command.
+
+`--python-package` optional CLI arg allows you to select a parent package
+name used by one generated file to import symbols from another one.
+
+If you **don't intend to embed** the KSC-generated files and just want to
+play a bit in IPython/Jupyter shell in the same dir, **remove** this arg,
+otherwise you get an `ImportError` .
+
+If you **intend to embed** the compilation result into an own package,
+it is recommended **to set the arg to `.`**, which means "current package",
+rather than to the real  name of the subpackage, because setting it
+to a real name will cause breakage when you (or anyone else) want
+to reorganize or rename your package. It will cause KSC to generate
+relative imports instead of absolute ones for the specs imported into
+your main spec:
+
+----
+from .your_imported_spec import YourImportedSpec
+----
+
+(note the dot we have passed as an arg) instead of
+
+----
+from your_imported_spec import YourImportedSpec
+----
+.
+
+**Without setting this arg at all you will likely get an `ImportError`,
+if any of the specs imports another spec.** You can instead set this
+arg to the absolute path of the submodule, but it violates DRY principle
+and will only result in additional work when refactoring or making an 
+independent fork of your lib.
+
+It is also recommended to put all the KSC-generated code into an
+own subpackage, in example `kaitai`, so, if your build process
+regenerates the files on every build, you could just .gitignore that dir.
 
-TODO: document `--python-package`
 
-[[add-runtime]]
-=== Adding runtime library
+[[install-runtime]]
+=== Installing runtime library
 
 Code generated by Kaitai Struct compiler will require additional runtime
-library (https://pypi.org/project/kaitaistruct/[kaitaistruct at PyPi],
+library (https://pypi.org/project/kaitaistruct/[image:https://img.shields.io/pypi/v/kaitaistruct.svg[kaitaistruct at PyPi]],
```

Comment:
Minor thing: it's called "PyPI", not "PyPi".

```suggestion
library (https://pypi.org/project/kaitaistruct/[image:https://img.shields.io/pypi/v/kaitaistruct.svg[kaitaistruct at PyPI]],
```

Reply:
Thanks, fixed.